### PR TITLE
Merge INSTALL.md and README.md and rework READMEs

### PR DESCRIPTION
**Pull request by gabriel-farache**
Closes [FLPATH-1718](https://issues.redhat.com/browse/FLPATH-1718)

Merge INSTALL.md and README.md and rework READMEs

Needs https://github.com/parodos-dev/serverless-workflows/pull/432 to be merged first as there are links to those README

I voluntarily did not change anything related to MTA/MTA v6 as our support is for MTA v7 only now

### CODE REVIEW

**Comment by ElaiShalevRH** [comment]
this is just taking INSTALL.md and inserting in README, correct?
loos ok

**Comment by gabriel-farache** [comment]
and see https://github.com/parodos-dev/serverless-workflows/pull/432

**Comment by gabriel-farache** [comment]
> this is just taking INSTALL.md and inserting in README, correct?

@ElaiShalevRH Yes and adding a section linking to the workflow's README for the configurable properties/env var

**Review by masayag — APPROVED** [review]

**Comment by masayag** [comment]
@gabriel-farache pls verify the parodos.dev refers to the correct paths.
we might require creating a branch for this repo for v1.2.x and refer to it from gh-pages-1.2
and/or from the website.
Also, aren't we having some duplication between gh-pages (and gh-pages-1.2) branch to main branch?

**Comment by gabriel-farache** [comment]
@masayag https://www.parodos.dev/1.2/docs/installation/workflows/deploy-from-helm-repository/ points to https://github.com/parodos-dev/serverless-workflows-config/blob/gh-pages/docs/<workflow>/README.md 

But my changes are on the `main` branch, not `gh-pages` and this would annoys me to duplicate the READMEs.

I will move the content of the READMEs of the `main` to the  `gh-pages` branches (both of them) and then I will put a link to the `gh-pages` (the one reflecting the `main` branch) in it.

Was that what you meant?

**Comment by gabriel-farache** [comment]
* In [parodos-dev.github.io](https://github.com/parodos-dev/parodos-dev.github.io)
  * https://github.com/parodos-dev/parodos-dev.github.io/blob/main/content/main/docs/serverless-workflows/infrastructure/move2kube.md is pointing to https://github.com/parodos-dev/serverless-workflows/blob/main/move2kube/README.md?raw=true
    *  it's fine for meas the `main` should points to the latest README which is what is on the `main` of https://github.com/parodos-dev/serverless-workflows
  * https://github.com/parodos-dev/parodos-dev.github.io/blob/main/content/1.2/docs/serverless-workflows/infrastructure/move2kube.md is also pointing to https://github.com/parodos-dev/serverless-workflows/blob/main/move2kube/README.md?raw=true
    * that is not good, this one should points to `gh-pages-1.2`

But now, what is the use of `gh-pages` for https://github.com/parodos-dev/serverless-workflows? If the source of truth is [parodos-dev.github.io](https://github.com/parodos-dev/parodos-dev.github.io) then we should not have any other website. Or if we need to have a website for the helm charts, then any doc related to workflow/use cases, should points to the source of truth website

File: charts/move2kube/README.md [hunk 1]
```diff
@@ -24,8 +25,117 @@ The following table lists the configurable parameters of the Move2kube chart and
...
+
+#### Set `M2K_ROUTE`, `K_SINK` and `BACKSTAGE_NOTIFICATIONS_URL` for the Sonataflow CR
+
+The value for `BACKSTAGE_NOTIFICATIONS_URL` in the command below is using the current default value, if the name of the backstage deployment or its namespace does not match, please update the value with the correct value from your cluster.
+
+Run the following to set `K_SINK`, `MOVE2KUBE_URL` and `BACKSTAGE_NOTIFICATIONS_URL`environment variable in the workflow:
+```console
+BACKSTAGE_NOTIFICATIONS_URL=http://backstage-backstage.rhdh-operator
+BROKER_URL=$(oc -n ${TARGET_NS} get broker -o yaml | yq -r .items[0].status.address.url)
+oc -n ${TARGET_NS} patch sonataflow m2k --type merge -p '{"spec": { "podTemplate": { "container": { "env": [{"name": "BACKSTAGE_NOTIFICATIONS_URL": "'${BACKSTAGE_NOTIFICATIONS_URL}'"},{"name": "K_SINK", "value": "'${BROKER_URL}'"}, {"name": "MOVE2KUBE_URL", "value": "https://'${M2K_ROUTE}'"}]}}}}'
```
Comment (y-first) [inline charts/move2kube/README.md:139]: @gabriel-farache there is a type here, should be `{"name": "BACKSTAGE_NOTIFICATIONS_URL",  "value": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}`

File: charts/mta-v7.x/README.md [hunk 2]
```diff
@@ -14,6 +17,89 @@ The following table lists the configurable parameters of the Mta chart and their
...
+
+**Please note** that it may take several minutes for the MTA Operator to become available and for the route to be reachable.
+```console
+while [[ $retry_count -lt 5 ]]; do
+    oc -n openshift-mta get route mta && break || sleep 60
+    retry_count=$((retry_count + 1))
+done
+BACKSTAGE_NOTIFICATIONS_URL=http://backstage-backstage.rhdh-operator
+MTA_ROUTE=$(oc -n openshift-mta get route mta -o yaml | yq -r .spec.host)
+oc -n ${TARGET_NS} patch sonataflow mta-analysis-v7 --type merge -p '{"spec": { "podTemplate": { "container": { "env": [{"name": "BACKSTAGE_NOTIFICATIONS_URL": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}, {"name": "MTA_URL", "value": "https://'${MTA_ROUTE}'"}]}}}}'
```
Comment (y-first) [inline charts/mta-v7.x/README.md:90]: Same typo, should be `{"name": "BACKSTAGE_NOTIFICATIONS_URL",  "value": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}`

File: charts/move2kube/README.md (hunk 1 above)
Comment (gabriel-farache) [reply charts/move2kube/README.md:139]: Thx, the fix is part of https://github.com/parodos-dev/serverless-workflows-config/pull/846

File: charts/mta-v7.x/README.md (hunk 2 above)
Comment (gabriel-farache) [reply charts/mta-v7.x/README.md:90]: Thx, the fix is part of https://github.com/parodos-dev/serverless-workflows-config/pull/846

**Pull request merged**